#!/usr/bin/env python
PACKAGE = "hector_barrel_detection_nodelet"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_H_value", int_t, 0, "Minimum hue value", 100, 0, 255)
gen.add("max_H_value", int_t, 0, "Maximum hue value", 140, 0, 255)
gen.add("min_S_value", int_t, 0, "Minimum saturation value", 0, 0, 255)
gen.add("max_S_value", int_t, 0, "Maximum saturation value", 255, 0, 255)
gen.add("min_V_value", int_t, 0, "Minimum value value", 0, 0, 255)
gen.add("max_V_value", int_t, 0, "Maximum value value", 255, 0, 255)

exit(gen.generate(PACKAGE, "hector_barrel_detection_nodelet", "BarrelDetection"))
